conf.py

```diff
@@ -141,7 +141,7 @@
 # a list of builtin themes.
 # html_theme = 'sphinx_rtd_theme'
 html_theme = "conan"
-html_theme_path = conan.get_html_theme_path()
+html_theme_path = [conan.get_html_theme_path()]
```

Comment:
```suggestion
html_theme_path = conan.get_html_theme_path()
```
releated with the [cur_dir] change